webapp/client/src/components/FormNavButtons.js

```diff
@@ -0,0 +1,132 @@
+import PropTypes from "prop-types";
+import styled from "styled-components";
+
+const Row = styled.div`
+  margin-top: var(--spacing-09);
+`;
+
+const Button = styled.button`
```

Comment:
How do you decide if you export some part as its own component or just set it like this Button here? Will we just have one component per component in Nadine's library?

Reply:
Eventually, that's my goal. Right now, I'm working bottom up: make standalone components for what needs to be shared to implement the login form, keep the rest local to others until the need arises. This is an ideal, I've already deviated from that a little bit when porting things that are standalone components in the jinja macros. I'm trying to be pragmatic and I'm sure we'll revisit what should be its own component and what should be a local utility later on.